verify/verify-pylint.sh

```diff
@@ -20,13 +20,10 @@ set -o pipefail
 export PYLINTHOME=$(mktemp -d)
 pylint="$(dirname $0)/pylint_bin"
 
-$pylint \
-  scenarios/*.py \
-  jenkins/*.py \
-  jobs/*.py \
-  metrics/*.py \
-  queue-health/graph/graph.py \
-  queue-health/weekly_commit_stats.py \
-  triage/*.py \
-  experiment/*.py \
-  verify/*.py
+
+# TODO(krzyzacy): pylint gubernator/ and kettle/
```

Comment:
You need to ignore bazel-* too.

I think it's cleaner to just use a bash glob:

    shopt -s extglob globstar
    pylint !(gubernator|kettle|external|mungegithub|bazel-*)/**/*.py